source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -120,7 +123,8 @@ KX_GameObject::KX_GameObject(
       m_components(NULL),
       m_pInstanceObjects(NULL),
       m_pDupliGroupObject(NULL),
-      m_actionManager(NULL)
+      m_actionManager(NULL),
+	  m_pSavedPhysicsController(NULL)
```

Comment:
What's the goal of `m_pSavedPhysicsController` ? Please avoid the type in name convention : replace `m_p` by `m_`.